Add mappers' guild update newspost (#3304)
ppy/osu-wiki: news/2020-03-13-featured-artist-beatmap-updates-from-the-mappers-guild.md

---
layout: post
title: Featured Artist Beatmap Updates from the Mappers' Guild
date: 2020-03-13 00:00:00 +0000
---

There's a ton of new beatmaps, completed quests, and medals from the Mappers' Guild celebrating osu!'s featured artists!

![](/wiki/shared/news/banners/mappersguild.png)

Since the [previous update](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) a lot has been happening in the Guild, with over **20** quests completed, over **100** beatmaps ranked, and **16** new medals for you to earn, all tied to the rapidly growing Featured Artist library!

> For those who don't know yet, the Mappers' Guild is a way for mappers with three or more ranked beatmaps to earn rewards for creating beatmaps of songs from the Featured Artist library. If you're interested in the Guild, be sure to look at the [Mappers' Guild website](https://mappersguild.com/) for more information.

Reviewer: better to link to the mappers guild wiki page. it'll be added in #3280

Author: done did

> Recently there has also been some new developments in the Mappers' Guild system! The first is that now anyone can view the website and keep up with what Mappers' Guild members are making, so be sure to check it out if you are curious about what the mappers are working on. Next, the Guild now has solo quests available for members to take on by themselves, giving mappers the option to work alone if that's what they prefer. Lastly, with the return of the [Monthly Beatmapping Contests](https://osu.ppy.sh/help/wiki/Contests/Monthly_Beatmapping_Contest), guild members who participate can also earn points in the Mappers' Guild for doing so, which you can read more about on the linked wiki page.

Reviewer: "guild" isn't capitalized after the MBC wiki link, whereas everywhere else it is
i think it'd make more sense to leave it uncapitalized when it's just "guild" alone and capitalize when it's the title "Mappers' Guild"

- [Ranked map updates](#ranked-map-updates)

## <a id="finished-quests"></a>Finished quests

![Aether Realm header](https://assets.ppy.sh/artists/47/header.jpg)

For the **Aether Realm mini-pack (osu!taiko)** quest, the mappers had to create and rank at least 3 mapsets of songs by Aether Realm, each hosted by a different user.

This quest was completed by **[Raiden](https://osu.ppy.sh/users/2239480)**, **[Greenshell](https://osu.ppy.sh/users/8693851)**, **[LZD](https://osu.ppy.sh/users/224335)** and **[frukoyurdakul](https://osu.ppy.sh/users/7612550)**

- [Aether Realm - Tarot](https://osu.ppy.sh/beatmapsets/1030740#taiko/2155553) by [Raiden](https://osu.ppy.sh/users/2239480)
- [Aether Realm - The Devil](https://osu.ppy.sh/beatmapsets/1053572#taiko/2201603) by [LZD](https://osu.ppy.sh/users/224335)
- [Aether Realm - The Emperor](https://osu.ppy.sh/beatmapsets/1047197#taiko/2189025) by [Greenshell](https://osu.ppy.sh/users/8693851)

![Carpool Tunnel header](https://assets.ppy.sh/artists/42/header.jpg)

For the **Carpool Tunnel mini-pack** quests, the mappers had to create and rank at least 4 mapsets of songs by Carpool Tunnel, each hosted by a different user.

> The **osu!catch** version of this quest was completed by **[Jemzuu](https://osu.ppy.sh/users/7890134)**, **[Crowley](https://osu.ppy.sh/users/6341006)**, **[JierYagtama](https://osu.ppy.sh/users/7483452)**, **[Du5t](https://osu.ppy.sh/users/6053071)** and **[Dapuluous](https://osu.ppy.sh/users/8140944)**

Reviewer: when multiple modes have completed the same quest, it'd make more sense to order them in the way people expect (osu, taiko, catch, mania)

- [Loki - Quiet Village](https://osu.ppy.sh/beatmapsets/1050693#osu/2195976) hosted by [Gero](https://osu.ppy.sh/users/1467715)
- [Loki - Wizard's Tower](https://osu.ppy.sh/beatmapsets/1064248#osu/2228263) hosted by [Minorsonek](https://osu.ppy.sh/users/5377178)
- [Loki - Troll of the Forsaken Cave](https://osu.ppy.sh/beatmapsets/1052028#osu/2198578) hosted by [Cris-](https://osu.ppy.sh/users/6175280)
- [Loki - With Fire and Sword (2013)](https://osu.ppy.sh/beatmapsets/1088528#osu/2276018) hosted by [TheShadowOfDark](https://osu.ppy.sh/users/5795337)
- [Loki - The Victory is Ours](https://osu.ppy.sh/beatmapsets/1066365#osu/2232865) hosted by [Crissa](https://osu.ppy.sh/users/5405836)
- [Loki - Interlude](https://osu.ppy.sh/beatmapsets/1091243#osu/2281346) hosted by [Tatan](https://osu.ppy.sh/users/5646529)

![Mystery header](/wiki/shared/news/banners/mappersguild-mystery.jpg)

> For the **Mystery Electronic mini-pack (osu!taiko)** quest, the mappers had to create and rank at least 3 mapsets of unpublished electronic songs by one of our unreleased featured artists.

Reviewer: insert paragraph about how these maps were paired with the fa announcement for MYLK https://osu.ppy.sh/home/news/2019-12-31-new-featured-artist-mylk

This quest was completed by **[tasuke912](https://osu.ppy.sh/users/2774767)**, **[salchow](https://osu.ppy.sh/users/9738159)**, **[HiroK](https://osu.ppy.sh/users/4050738)**, **[Nepuri](https://osu.ppy.sh/users/6637817)**, **[Cynplytholowazy](https://osu.ppy.sh/users/3901754)** and **[Nardoxyribonucleic](https://osu.ppy.sh/users/876419)**

- [MYLK - Let You Go](https://osu.ppy.sh/beatmapsets/1053583#taiko/2201615) hosted by [Nepuri](https://osu.ppy.sh/users/6637817)
- [MYLK - Prism](https://osu.ppy.sh/beatmapsets/1051590#taiko/2197618) hosted by [salchow](https://osu.ppy.sh/users/9738159)
- [MYLK - Smile](https://osu.ppy.sh/beatmapsets/1065517#taiko/2231210) hosted by [Nardoxyribonucleic](https://osu.ppy.sh/users/876419)
- [MYLK - Bubblegum](https://osu.ppy.sh/beatmapsets/1055143) hosted by [Cynplytholowazy](https://osu.ppy.sh/users/3901754)
- [MYLK - SOAP](https://osu.ppy.sh/beatmapsets/1055618#taiko/2205809) hosted by [tasuke912](https://osu.ppy.sh/users/2774767)

![Mystery header](/wiki/shared/news/banners/mappersguild-mystery.jpg)

> For the **Mystery Rock mini-pack (osu!, osu!taiko, osu!catch)** quest, the mappers had to create and rank at least 3 mapsets of unpublished rock songs by one of our unreleased featured artists.

Reviewer: insert paragraph about how these maps were paired with the fa announcement for Blue Stahli https://osu.ppy.sh/home/news/2020-01-22-new-featured-artist-blue-stahli

This quest was completed by **[fieryrage](https://osu.ppy.sh/users/3533958)**, **[Kiryu Coco](https://osu.ppy.sh/users/4236057)**, **[MBomb](https://osu.ppy.sh/users/3071175)**, **[Cynplytholowazy](https://osu.ppy.sh/users/3901754)**, **[idke](https://osu.ppy.sh/users/4650315)** and **[salchow](https://osu.ppy.sh/users/9738159)**

- [Blue Stahli - Suit Up](https://osu.ppy.sh/beatmapsets/1061425/) by [MBomb](https://osu.ppy.sh/users/3071175) (osu!catch)
- [Blue Stahli - Throw Away](https://osu.ppy.sh/beatmapsets/1061411#osu/2222638) hosted by [fieryrage](https://osu.ppy.sh/users/3533958) (hybrid)
- [Blue Stahli and Nyzze Nyce - Run The Track](https://osu.ppy.sh/beatmapsets/1071229) hosted by [Kiryu Coco](https://osu.ppy.sh/users/4236057) (osu!catch)

![Native Construct header](https://assets.ppy.sh/artists/50/header.jpg)

> For the **Native Construct mini-pack** quests, the mappers had to create and rank at least 3 mapsets of songs by Native Construct. At least one mapset must be for Chromatic Abberation.

Reviewer: ```suggestion
For the **Native Construct mini-pack** quests, the mappers had to create and rank at least 3 mapsets of songs by Native Construct. At least one mapset had to be for Chromatic Aberration.
```

- [Nekrogoblikon - Mold](https://osu.ppy.sh/beatmapsets/1061153#osu/2221627) hosted by [Icekalt](https://osu.ppy.sh/users/5410645)
- [Nekrogoblikon - The Skin Thief](https://osu.ppy.sh/beatmapsets/1061535) hosted by [LMT](https://osu.ppy.sh/users/7262798)
- [Nekrogoblikon - Dragons](https://osu.ppy.sh/beatmapsets/1083931#osu/2266883) by [Nao Tomori](https://osu.ppy.sh/users/5364763)

![Panda Eyes header](https://assets.ppy.sh/artists/55/header.jpg)

For the **On My Own (osu!taiko)** quest, the mappers had to create and rank beatmap for "On My Own" by Panda Eyes

This quest was completed by **[Cynplytholowazy](https://osu.ppy.sh/users/3901754)** and **[Faputa](https://osu.ppy.sh/users/845733)**

- [Panda Eyes - On My Own](https://osu.ppy.sh/beatmapsets/1065837) hosted by [Cynplytholowazy](https://osu.ppy.sh/users/3901754)

For the **Panda Eyes solo** quests, each mapper had to create and rank 1 mapset in their mode of any song by Panda Eyes

> The **osu!catch** version of this quest was completed by **[Nelly](https://osu.ppy.sh/users/4741164)**

Reviewer: when multiple modes have completed the same quest, it'd make more sense to order them in the way people expect (osu, taiko, catch, mania)

The **osu!** version of this quest was completed by **[Hazu-](https://osu.ppy.sh/users/4668230)**

- [Panda Eyes - Pump It Up](https://osu.ppy.sh/beatmapsets/1090677#osu/2280138) by [Hazu-](https://osu.ppy.sh/users/4668230)

![Ricky Montgomery header](https://assets.ppy.sh/artists/54/header.jpg)

For the **Ricky Montgomery mini-pack (osu!)** quest, the mappers had to create and rank at least 3 mapsets of songs by Ricky Montgomery

This quest was completed by **[Trynna](https://osu.ppy.sh/users/2652951)** and **[eiri-](https://osu.ppy.sh/users/3388410)**

- [Ricky Montgomery - Cabo](https://osu.ppy.sh/beatmapsets/1081806) hosted by [Trynna](https://osu.ppy.sh/users/2652951)
- [Ricky Montgomery - California](https://osu.ppy.sh/beatmapsets/1085943#osu/2270844) hosted by [eiri-](https://osu.ppy.sh/users/3388410)
- [Ricky Montgomery - Snow](https://osu.ppy.sh/beatmapsets/1086534#osu/2272128) hosted by [eiri-](https://osu.ppy.sh/users/3388410)

For the **Ricky Montgomery solo** quests, each mapper had to create and rank 1 mapset in their mode of any song by Ricky Montgomery

> The **osu!taiko** version of this quest was completed by **[Faputa](https://osu.ppy.sh/users/845733)**

Reviewer: when multiple modes have completed the same quest, it'd make more sense to order them in the way people expect (osu, taiko, catch, mania)

- [Wisp X - Frozen Leaves](https://osu.ppy.sh/beatmapsets/871914#osu/1822068) by [greenhue](https://osu.ppy.sh/users/4967662)
- [Wisp X - Hibiscus](https://osu.ppy.sh/beatmapsets/1040161#osu/2174050) by [Aeril](https://osu.ppy.sh/users/4334976)

![onumi header](https://assets.ppy.sh/artists/32/header.jpg)

For the **onumi mini-pack (osu!)** quest, the mappers had to create and rank at least 4 mapsets of songs by onumi, each hosted by a different user.

This quest was completed by **[PandaHero](https://osu.ppy.sh/users/1233255)**, **[Hectic](https://osu.ppy.sh/users/6885942)**, **[Mirash](https://osu.ppy.sh/users/2841009)** and **[Leader](https://osu.ppy.sh/users/631530)**

- [onumi - NOSTALGIA](https://osu.ppy.sh/beatmapsets/952393#osu/1988724) hosted by [PandaHero](https://osu.ppy.sh/users/1233255)
- [onumi - ARROGANCE](https://osu.ppy.sh/beatmapsets/940377) hosted by [Mirash](https://osu.ppy.sh/users/2841009)
- [onumi - CONFUSION PART ONE](https://osu.ppy.sh/beatmapsets/1037665#osu/2169357) hosted by [Leader](https://osu.ppy.sh/users/631530)
- [onumi - DESIRE](https://osu.ppy.sh/beatmapsets/1043754#osu/2182173) hosted by [Hectic](https://osu.ppy.sh/users/6885942)

> ## <a id="#pack-medal-updates"></a>Pack and medal updates

Reviewer: ```suggestion
## <a id="pack-medal-updates"></a>Pack and medal updates
```

> ## <a id="#ranked-map-updates"></a>Ranked map updates

Reviewer: ```suggestion
## <a id="ranked-map-updates"></a>Ranked map updates
```

- [MYLK - Let You Go](https://osu.ppy.sh/beatmapsets/1053583#taiko/2201615) hosted by [Nepuri](https://osu.ppy.sh/users/6637817)
- [MYLK - Prism](https://osu.ppy.sh/beatmapsets/1051590#taiko/2197618) hosted by [salchow](https://osu.ppy.sh/users/9738159)
- [MYLK - Smile](https://osu.ppy.sh/beatmapsets/1065517#taiko/2231210) hosted by [Nardoxyribonucleic](https://osu.ppy.sh/users/876419)
- [MYLK - Bubblegum](https://osu.ppy.sh/beatmapsets/1055143) hosted by [Cynplytholowazy](https://osu.ppy.sh/users/3901754)
- [MYLK - SOAP](https://osu.ppy.sh/beatmapsets/1055618#taiko/2205809) hosted by [tasuke912](https://osu.ppy.sh/users/2774767)

![Mystery header](/wiki/shared/news/banners/mappersguild-mystery.jpg)

For the **Mystery Rock mini-pack (osu!, osu!taiko, osu!catch)** quest, the mappers had to create and rank at least 3 mapsets of unpublished rock songs by one of our unreleased featured artists. These maps were created in secret by guild members to accompany the [Blue Stahli Featured Artist Announcement](https://osu.ppy.sh/home/news/2020-01-22-new-featured-artist-blue-stahli) this past January.

> This quest was completed by **[fieryrage](https://osu.ppy.sh/users/3533958)**, **[Kiryu Coco](https://osu.ppy.sh/users/4236057)**, **[MBomb](https://osu.ppy.sh/users/3071175)**, **[Cynplytholowazy](https://osu.ppy.sh/users/3901754)**, **[idke](https://osu.ppy.sh/users/4650315)** and **[salchow](https://osu.ppy.sh/users/9738159)**

Reviewer 2: can u change my names back to sinnoh ty
